runtime/template/v2/esi_custom.go

```diff
@@ -0,0 +1,29 @@
+// /*
+// Copyright © 2025 ESO Maintainer Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// */
+
+// Copyright External Secrets Inc
+// All Rights reserved.
+
+// Package template implements the template engine for External Secrets.
+package template
+
+import "encoding/json"
+
+// Verifies if given string is a valid json object.
+func isJSON(str string) bool {
+	var js map[string]interface{}
+	return json.Unmarshal([]byte(str), &js) == nil
+}
```

Comment:
This will not work with pure lists like `[1,2,3]` or `null` which are both valid JSONs.